tests/utils.go

```diff
@@ -100,3 +110,38 @@ func GetKubeClientFromRESTConfig(config *rest.Config) (*kubernetes.Clientset, er
 
 	return coreClient, nil
 }
+
+// Creates a new namespace with a randomly generated name that starts with cdi-test-
+func GenerateNamespace(client *kubernetes.Clientset) *k8sv1.Namespace {
```

Comment:
may want to pass in a base ns name which is prefixed to the generated ns name. See [`CreateTestingNS`](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/util.go#L1010)
Also, I don't know if this applicable to cdi but when k8s e2e generates a ns they also call [WaitForDefaultServiceAccountInNamespace](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/util.go#L1039)

Reply:
I am trying to keep this as simple as possible, we can always add the check for the service account later if it turns out to be a problem. But from my initial inspection I don't think it will be. Our tests are significantly simpler than the k8s ones.

I will add prefix, that way when we get the Framework in place, we can take the name passed in (the suite name or whatever) and use that to make the NS even more unique.